### DISCUSSION TITLE
What does adding `~/kit/bin` to PATH do?

### DISCUSSION THREAD

**Discussion by stevebarakat** [johnlindquist/kit#448] [Q&A] [answered]
Adding `~/kenv/bin` to PATH allows me to run script from any directory. What does adding `~/kit/bin` to PATH do?

**Comment by johnlindquist** [comment, marked as answer]
It allows you to run `kit` and `sk` from any directory:

* `kit` is a tool to manage and run scripts from the terminal
* `sk` allows you to invoke the app from the terminal. `sk my-script` would launch the `my-script` in the app prompt and wait for user input.
